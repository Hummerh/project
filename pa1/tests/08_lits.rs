true   false   321   566974   1 2 3 4 5 6 7 8 9 10

