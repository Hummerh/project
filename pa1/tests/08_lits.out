LITBOOL(true)
LITBOOL(false)
LITDEC(321)
LITDEC(566974)
LITDEC(1)
LITDEC(2)
LITDEC(3)
LITDEC(4)
LITDEC(5)
LITDEC(6)
LITDEC(7)
LITDEC(8)
LITDEC(9)
LITDEC(10)
